tests/integration/create_cluster/ha_gce/expected-v1alpha2.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   creationTimestamp: "2017-01-01T00:00:00Z"
   name: ha-gce.example.com
 spec:
+  GCEServiceAccount: default
```

Comment:
needs to be lowercased (gceServiceAccount)

Reply:
will get there eventually :)